go/vt/sqlparser/sql.y

```diff
@@ -9086,6 +9333,7 @@ non_reserved_keyword:
 | DESCRIPTION
 | DIRECTORY
 | DISABLE
+| DISCARD
```

Comment:
Did all the new keywords from `keywords.go` get added to these keyword sections? I did a quick search in my local vitess project and didn't see others, like `COPY`, `EXCLUSIVE`, etc in the lists yet, but my local checkout may be a little out of date.

Reply:
Might just be out of date, copy is [here](https://github.com/dolthub/vitess/pull/342/files#diff-39206762b8e3a9651f705287954fc0a07c6517048c936937e36eff1215d41c18R157) and exclusive is [here](https://github.com/dolthub/vitess/pull/342/files#diff-39206762b8e3a9651f705287954fc0a07c6517048c936937e36eff1215d41c18R232)